src/main/java/io/smartcat/migration/MigrationEngine.java

```diff
@@ -55,18 +59,22 @@ public boolean migrate(final MigrationResources resources) {
             for (final Migration migration : resources.getMigrations()) {
                 final MigrationType type = migration.getType();
                 final int migrationVersion = migration.getVersion();
-                final int version = versioner.getCurrentVersion(type);
+                final int version = versioner.getCurrentVersion(type, ONE);
 
                 LOGGER.info("Db is version {} for type {}.", version, type.name());
                 LOGGER.info("Compare {} migration version {} with description {}", type.name(), migrationVersion,
-                        migration.getDescription());
+                            migration.getDescription()
+                );
 
                 if (migrationVersion <= version) {
                     LOGGER.warn("Skipping migration [{}] with version {} since db is on higher version {}.",
-                            migration.getDescription(), migrationVersion, version);
+                                migration.getDescription(), migrationVersion, version
```

Comment:
Is this formatting from the project formatter configuration?

Reply:
All of the formatting changes are reverted. Formatter was not configured properly.